cloud-sql/mysql/activerecord/config/puma.rb

```diff
@@ -17,7 +17,7 @@
 environment ENV.fetch("RAILS_ENV") { "development" }
 
 # Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
```

Comment:
Deploying to App Engine would fail with the error:

> bundler: failed to load command: rackup (/usr/local/bundle/bin/rackup)
> /usr/local/bundle/gems/puma-3.12.6/lib/puma/launcher.rb:133:in `initialize': No such file or directory @ rb_sysopen - tmp/pids/server.pid (Errno::ENOENT)

Updating `pumb.rb` to match the Postgres sample resolved this error.

>should we be doing this for other samples too?

The other samples already did not include this line.